stdlib/testing/diff.go

```diff
@@ -109,6 +109,7 @@ type DiffTransformation struct {
 
 	d     execute.Dataset
 	cache execute.TableBuilderCache
+	alloc *memory.Allocator
```

Comment:
Where does the `alloc` field get set?

Reply:
nevermind I see it now, somehow I missed on first review

Reply:
It gets set in `NewDiffTransformation`.